api/commands/validate.ts

```diff
@@ -0,0 +1,50 @@
+import * as cronParser from "cron-parser";
+import { dataform } from "@dataform/protos";
+
+
+function validateEmail(email: string) {
+  const email_regex = /^(([^<>()\[\]\.,;:\s@\"]+(\.[^<>()\[\]\.,;:\s@\"]+)*)|(\".+\"))@(([^<>()[\]\.,;:\s@\"]+\.)+[^<>()[\]\.,;:\s@\"]{2,})$/i;
```

Comment:
prefer camelCase for variables in general, here and in a few other places in the PR